ivy_tests/test_ivy/test_frontends/test_paddle/test_manipulation.py

```diff
@@ -721,6 +721,43 @@ def test_paddle_tile(
     )
 
 
+# unbind
+@handle_frontend_test(
+    fn_tree="paddle.unbind",
+    dtypes_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        min_num_dims=2,
+        max_num_dims=2,
```

Comment:
May be we could increase the number here so that arrays with more dimensions are also included in testing.